components/nimbus/src/lib.rs

```diff
@@ -161,10 +173,11 @@ impl NimbusClient {
     }
 
     pub fn get_available_experiments(&self) -> Result<Vec<AvailableExperiment>> {
+        let state = self.mutable_state.lock().unwrap();
```

Comment:
**IMPORTANT NOTE**: Will this API ever be called on the main thread? Note for myself to check, we shouldn't lock the main thread.

Reply:
Not relevant anymore, the app context is no longer mutable